muikku-school-data-pyramus/src/main/java/fi/otavanopisto/muikku/plugins/schooldatapyramus/PyramusUserSchoolDataBridge.java

```diff
@@ -1180,6 +1181,21 @@ public boolean isActiveUser(User user) {
     return true;
   }
   
+
+  @Override
+  public BridgeResponse<List<OrganizationContactPerson>> listOrganizationContactPersonsByOrganization(
+      Long organizationId) {
```

Comment:
This will change into `String organizationIdentifier` so turn the identifier string into Pyramus' organizationId using `identifierMapper`.